### ISSUE TITLE
5. 神经网络 > 3. 通过PyTorch进行深度学习 > 【4.定义网络】> 开头的提示没有看懂

### ISSUE THREAD

**Issue by dadatangtuan** [udacity/aipnd-issues-zh#479]
## 课程导航+地址
5. 神经网络 > 3. 通过PyTorch进行深度学习 > 【4.定义网络】
https://classroom.udacity.com/nanodegrees/nd089-cn/parts/4bfcd28a-ad62-48a0-ad9e-c346068df68e/modules/47a93218-8271-457b-aca6-e2cdca10131b/lessons/23055792-a4f4-4ed7-aa59-3fe7516c2fcc/concepts/b21f290b-6ef8-4679-851b-a38245d477d8#

## 截图
![image](https://user-images.githubusercontent.com/41724066/43999905-aff15ed4-9e48-11e8-915b-5b03f26a07eb.png)

## 具体描述
开头第一条文本
1、希望能直接附上相关视频课程的链接，不用大家再回头查找
2、前两个句号内的内容能明白在讲什么。最后部分”显示 nn.Linear、F.relu、nn.dropout。“加在其中，不太清楚想传递什么意思。

**Comment by DrTrippleJ** [comment]
@UdacityShuwenLiu 这里中文版的和英文版的差个视频

**Comment by UdacityShuwenLiu** [comment]
已更新
